packages/web-components/src/menu-item/menu-item.styles.ts

```diff
@@ -0,0 +1,203 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorCompoundBrandForeground1Hover,
+  colorCompoundBrandForeground1Pressed,
+  colorNeutralBackground1,
+  colorNeutralBackground1Hover,
+  colorNeutralBackground1Selected,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForeground2,
+  colorNeutralForeground2Hover,
+  colorNeutralForeground2Pressed,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeDisabled,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase500,
+  fontWeightRegular,
+  lineHeightBase200,
+  lineHeightBase300,
+} from '../theme/design-tokens.js';
+
+/** Menu styles
+ * @public
+ */
+export const styles = css`
+  ${display('grid')}
+
+  :host {
+    grid-template-columns: 20px 20px auto 20px;
+    align-items: center;
+    grid-gap: 4px;
+    height: 32px;
+    background: ${colorNeutralBackground1};
+    font: ${fontWeightRegular} ${fontSizeBase300} / ${lineHeightBase300} ${fontFamilyBase};
+    border-radius: ${borderRadiusMedium};
+    padding: 0 10px;
+    margin: 1px 0;
```

Comment:
We typically avoid baking margin into the `host`

Reply:
Yeah, IMO components should not involve themselves with things _outside_ their bounds. This is essentially what this is doing, it's spacing it between other items which is typically applied _to_ an element rather than _by_ an element - what's this intended to do?

Reply:
removed